scripts/utils/cliCommandWrapper.ts

```diff
@@ -64,7 +64,11 @@ export const cliCommandWrapper = async <ArgsSchema extends z.Schema | undefined>
 
   let isSuccess = true
   try {
-    await command(app.diContainer.cradle, reqContext, args)
+    await command(
+      app.diContainer.cradle,
+      reqContext,
+      args as ArgsSchema extends z.Schema ? z.infer<ArgsSchema> : undefined,
```

Comment:
I'd use either z.output or z.input, depending on intention, to make it more explicit